src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
```

Comment:
The variable name 'sourceArray' is not descriptive of its contents. Consider renaming it to reflect that it contains CSS style declarations.